src/query/reverse_deps.go

```diff
@@ -1,24 +1,23 @@
 package query
 
 import (
+	"core"
 	"fmt"
 	"sort"
-
-	"core"
 )
 
 // ReverseDeps For each input label, finds all targets which depend upon it.
-func ReverseDeps(graph *core.BuildGraph, labels []core.BuildLabel) {
-
-	uniqueTargets := make(map[core.BuildLabel]struct{})
+func ReverseDeps(state *core.BuildState, labels []core.BuildLabel) {
+	uniqueTargets := make(map[core.BuildLabel]*core.BuildTarget)
```

Comment:
I don't know the labels are very useful (the target has them anyway); might be worth turning this into a `map[*core.BuildTarget]struct{}`

Reply:
Good point, yeah